packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,217 @@
+import type { ReactChild, ReactElement } from 'react';
+import { useState } from 'react';
+
+import StyledAlert from './StyledAlert';
+import { AlertIcon, CloseIcon } from '~components/Icons';
+import { makeAccessible } from '~utils';
+import { IconButton } from '~components/Button/IconButton';
+import Box from '~components/Box';
+import { Heading, Text } from '~components/Typography';
+import BaseButton from '~components/Button/BaseButton';
+import { BaseLink } from '~components/Link/BaseLink';
+
+type Nullable<Type> = Type | null;
+
+type PrimaryAction = {
+  text: string;
+  onClick: () => void;
+};
+
+type SecondaryActionButton = {
+  text: string;
+  onClick: () => void;
+};
+
+type SecondaryActionLinkButton = {
+  text: string;
+  href: string;
+  onClick?: () => void;
+  target?: string;
+  /**
+   * When `target` is set to `_blank` this is automatically set to `noopener noreferrer`
+   */
+  rel?: string;
+};
+
+type SecondaryAction = SecondaryActionButton | SecondaryActionLinkButton;
+
+export type AlertProps = {
+  /**
+   * Body content, pass text or JSX. Avoid passing components except `Link` to customize the content.
+   */
+  description: ReactChild;
+
+  /**
+   * A brief heading
+   */
+  title?: string;
+
+  /**
+   * Shows a dismiss button
+   *
+   * @default true
+   */
+  isDismissable?: boolean;
+
+  /**
+   * A callback when the dismiss button is clicked
+   */
+  onDismiss?: () => void;
+
+  /**
+   * Can be set to `high` for a more prominent look. Not to be confused with a11y contrast.
+   *
+   * @default low
+   */
+  contrast?: 'high' | 'low';
```

Comment:
Let's use `ColorContrastTypes` if we inline these we have to make change in future all over the places if this union changes. 

```suggestion
  contrast?: ColorContrastTypes;
```

Reply:
Sure, kept it inline because it shows up nicely in text editor type hints. Otherwise it would just show `ColorContrastTypes`. [Related](https://razorpay.slack.com/archives/G01B3LQ9H0W/p1660723807658419).

Also in future if we make changes in contrast then we'll have to thoroughly make changes in components anyway.

Reply:
Done in https://github.com/razorpay/blade/pull/654/commits/ee1c1bbada067d0954583da2d27b9ffd898dee65